### PR TITLE
restart first commit

### PR DESCRIPTION
**Pull request by Bela-Szilagyi**
Please review!

### CODE REVIEW

File: src/com/codecool/snake/Game.java [hunk 1]
```diff
@@ -3,8 +3,11 @@
 import com.codecool.snake.entities.enemies.SimpleEnemy;
 import com.codecool.snake.entities.powerups.SimplePowerup;
 import com.codecool.snake.entities.snakes.SnakeHead;
+import javafx.event.EventHandler;
 import javafx.scene.Scene;
 import javafx.scene.layout.Pane;
+import javafx.scene.input.MouseEvent;
```
Comment (bekor) [inline src/com/codecool/snake/Game.java]: Not used event

File: src/com/codecool/snake/Snake.java [hunk 2]
```diff
@@ -1,19 +1,46 @@
 package com.codecool.snake;
 
+import com.codecool.snake.entities.GameEntity;
+import com.codecool.snake.entities.snakes.SnakeHead;
 import javafx.application.Application;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
 import javafx.scene.Scene;
+import javafx.scene.input.MouseEvent;
```
Comment (bekor) [inline src/com/codecool/snake/Snake.java]: Not used event

**Review by bekor — COMMENTED** [review]
Some java package not used

**Review by Bela-Szilagyi — COMMENTED** [review]
MouseEvents deleted.

**Pull request merged**